WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import WooCommerce
+
+final class PointOfSaleDashboardViewModelTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func test_viewModel_empty_cart() {
+        let viewModel = PointOfSaleDashboardViewModel(items: POSItemProviderPreview().providePointOfSaleItems(),
+                                                      cardPresentPaymentService: CardPresentPaymentPreviewService())
+
+        XCTAssertTrue(viewModel.itemsInCart.isEmpty)
```

Comment:
nit: to follow the test method naming convention https://github.com/woocommerce/woocommerce-ios/blob/50b9bffa95aeac300c5f9fdd3d6f79c7f111a326/docs/naming-conventions.md#test-methods, it'd be great to include more details in the method name

```suggestion
    func test_cart_is_empty_initially() {
        // Given
        let viewModel = PointOfSaleDashboardViewModel(items: POSItemProviderPreview().providePointOfSaleItems(),
                                                      cardPresentPaymentService: CardPresentPaymentPreviewService())

        // Then
        XCTAssertTrue(viewModel.itemsInCart.isEmpty)
```